blueprints/codes/cur/cur_228/formula_2_22.py

```diff
@@ -0,0 +1,68 @@
+"""Contains formula 2.22 from CUR 228."""
+
+from blueprints.codes.cur.cur_228 import CUR_228
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula
+from blueprints.type_alias import KN_M3, KPA, M
+
+
+class Form2Dot22MenardStiffness(Formula):
+    """Representation of equation 2.22 CUR 228."""
+
+    source_document = CUR_228
+    label = "2.22"
+
+    def __init__(self, r: M, e_p: KPA, alpha: float) -> None:
```

Comment:
Use dimensionless instead of float

```suggestion
from blueprints.type_alias import KN_M3, KPA, M, DIMENSIONLESS


class Form2Dot22MenardStiffness(Formula):
    """Representation of equation 2.22 CUR 228."""

    source_document = CUR_228
    label = "2.22"

    def __init__(self, r: M, e_p: KPA, alpha: DIMENSIONLESS) -> None:
```